python/grass/temporal/univar_statistics.py

```diff
@@ -65,12 +65,9 @@ def compute_univar_stats(registered_map_info, stats_module, fs, rast_region=Fals
 
     if not univar_stats:
         gs.warning(
-            _(
-                "Unable to get statistics for {voxel}raster map "
-                "<{rmap}>".format(
-                    rmap=id, voxel="" if stats_module.name == "r.univar" else "3d "
-                )
-            )
+            _("Unable to get statistics for raster map <%s>") % id
+            if stats_module.name == "r.univar"
+            else _("Unable to get statistics for 3d raster map <%s>") % id
```

Comment:
We use uppercase D in 3D in sentences, e.g., 3D view in GUI.

```suggestion
            else _("Unable to get statistics for 3D raster map <%s>") % id
```

Reply:
Can we do that correction after this PR is merged and translation files picked up the changes? The source string (that wasn't found anymore in English, but exists in 4 translated languages) had lowercase 3d.

There's more of my original improvements to go back after that anyways.

Reply:
If you think it's better that way, sure!